docs/getting-started/quickstart.rst

```diff
@@ -1,39 +1,19 @@
 .. _quickstart-setup:
 
-Quickstart Setup
-----------------
+Recommended Setup
+-----------------
 
-The best way to get started and learn Gammapy are the :ref:`tutorials`. For
-convenience we provide a pre-defined conda environment file, so you can
-get additional useful packages together with Gammapy in a virtual isolated
-environment. First install `Miniconda <https://docs.conda.io/en/latest/miniconda.html>`__
-and then just execute the following commands in the terminal:
+We recommend using :ref:`virtual environments <virtual-envs>`, to do so
+execute the following commands in the terminal:
 
 .. substitution-code-block:: bash
 
     $ curl -O https://gammapy.org/download/install/gammapy-|release|-environment.yml
     $ conda env create -f gammapy-|release|-environment.yml
 
-.. note::
-
-    On Windows, you have to open up the conda environment file and delete the
-    lines with ``sherpa`` and ``healpy``. Those are optional dependencies that
-    currently aren't available on Windows.
-
-.. note::
-
-    For Apple silicon M1 (`arm64`) architectures you also have to open the
-    environment file and delete the `sherpa` entry, as currently there are
-    no conda packages available. However you can later install `sherpa`
-    in the environment using `python -m pip install sherpa`.
-
-Once the environment has been created you can activate it using:
-
-.. substitution-code-block:: bash
-
-    $ conda activate gammapy-|release|
```

Comment:
Activating the environment needs to happen first, otherwise the later download will fail, because the `gammapy` command is not available.